test/user-profile/helpers.unit.spec.js

```diff
@@ -76,7 +76,7 @@ describe('profile helpers:', () => {
       const allMappedIds = Object.keys(schemaToConfigIds);
       const sipEnabledConfigs = configs.filter(config => !config.disableSave);
       const sipEnabledFormIds = sipEnabledConfigs.map(sipEnabledConfig => sipEnabledConfig.formId);
-      expect(allFormIds).to.deep.equal(allMappedIds);
+      expect(allFormIds).to.include.members(allMappedIds);
```

Comment:
@jbalboni LMK what you think about this. I think there was a placeholder included in the `vets-json-schema` that's not implemented yet in `vets-website` and since I added a change after this commit things are out of sync. I think this check now doesn't fail if there's something in the `vets-json-schema` that isn't implemented in `vets-website`.

Reply:
I think this is ok, but I would look and see if adding that net form to the ignored forms Set in the file this is testing helps. @cehsu may remember more about these tests than me.

Reply:
Good idea- I'll update the unit tests